modules/odc-grouping-multiple-components.adoc

```diff
@@ -15,23 +15,22 @@ The following procedure adds a MongoDB database service to an existing applicati
 
 .Procedure
 
-. Create and deploy the MongoDB service to your project as follows:
-.. In the *Developer* perspective, navigate to the *Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
-.. Click on the *MongoDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MongoDB service, and click *Create* to create the service.
+. Create and deploy the MariaDB service to your project as follows:
+.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all of the available services in the *Developer Catalog*.
+.. Select *Databases* to list all the database services, and then click on the *MariaDB* option to see the details for the service.
+.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
 
-. On the left navigation panel, click *Topology* to see the MongoDB service deployed in your project.
-. To add the MongoDB service to the existing application group, select the *mongodb* pod and drag it to the application; the MongoDB service is added to the existing application group.
-. Dragging a component and adding it to an application group automatically adds the required labels to the component. Click on the MongoDB service node to see the label `app.kubernetes.io/part-of=myapp` added to the *Labels* section in the *Overview* Panel.
+. To add the MariaDB service to the existing application group, kbd:[Shift]+ dragging the *mariadb* pod to the application group containing the Node.Js component; the MariaDB service is added to the existing application group.
+. Click on the MariaDB service node to see the label `app.kubernetes.io/part-of=<application-name>` added to the *Labels* section in the *Overview* Panel. kbd:[Shift]+ dragging a component and adding it to an application group automatically adds the required labels to the component.
 +
 .Application grouping
 image::odc_app_grouping_label.png[]
 
-Alternatively, you can also add the component to an application as follows:
+Alternatively, you can also add a component in the *Topology* view to an application as follows:
```

Comment:
This information shouldn't be at the bottom. It should be presented as an alternate set of substeps around line 23.